README.rst

```diff
@@ -3,13 +3,19 @@ Dask-XGBoost
 
 Distributed training with XGBoost and Dask.distributed
 
-This repository enables you to perform distributed training with XGBoost on
+This repository offers one option to perform distributed training with XGBoost on
 Dask.array and Dask.dataframe collections.
 
 ::
 
    pip install dask-xgboost
 
+Please note that XGBoost also includes a Dask API as part of its official Python package.
+That API is independent of `dask-xgboost` and is the alternative recommended by the core XGBoost
```

Comment:
I think we can be a bit stronger here and say that it's *the* recommended way to use Dask & XGBoost together, not just by the XGBoost maintainers.

So I'd recommend dropping "alternative" and "core XGBoost development team" in favor of "the recommended way to use Dask and XGBoost together".

Reply:
Thanks, @TomAugspurger ! I updated the wording based on your suggestions.